test/scripts/agent-images/create_agent_images.sh

```diff
@@ -196,7 +196,7 @@ build_qcow2_image() {
                     quay.io/centos-bootc/bootc-image-builder:latest \
                     build \
                     --type qcow2 \
-                    --local "${REGISTRY_ADDRESS}/flightctl-device:base"
+                    "${REGISTRY_ADDRESS}/flightctl-device:base"
```

Comment:
The logs had this warning as --local is the default and it's not needed anymore.

```
sudo podman run --rm -it --privileged --pull=newer --security-opt label=type:unconfined_t -v /home/camadorg/dev/flight-control/flightctl/bin/output:/output -v /home/camadorg/dev/flight-control/flightctl/bin/dnf-cache:/var/cache/dnf:Z -v /home/camadorg/dev/flight-control/flightctl/bin/osbuild-cache:/var/cache/osbuild:Z -v /var/lib/containers/storage:/var/lib/containers/storage quay.io/centos-bootc/bootc-image-builder:latest build --type qcow2 --local 192.168.1.132:5000/flightctl-device:base
WARNING: --local is now the default behavior, you can remove it from the command line
```